plugintest/snat_test.go

```
@@ -0,0 +1,108 @@
+		})
+
+		It("snat SetSnat", func() {
+			cmd := plugin.RemoveSnatCmd{
+				NatInfo: []plugin.SnatInfo{env.snat1, env.snat2},
+			}
+			plugin.RemoveSnat(&cmd)
+			b := utils.Bash{
+				Command: "iptables-save -t nat",
+				Sudo:    true,
+			}
+			b.Run()
```

golangci-lint

Error return value of `b.Run` is not checked

(errcheck)